drivers/watchdog/watchdog_advantech.c

```diff
@@ -273,8 +273,10 @@ static long adv_wdt_ioctl(struct file *file, unsigned int cmd,
 	case WDIOC_SETTIMEOUT:
 		if (get_user(new_value, p))
 			return -EFAULT;
+
 		if ((new_value < 1) || (new_value > ADV_WDT_MAX_TIME))
 		{
+			pr_err("timeout value must be between 1 and %d\n", ADV_WDT_MAX_TIME);
```

Comment:
@advcharles No watchdog iotcl for max timeout value. So, I add a error prompt for timeout range.